platform/features/timeline/bundle.js

```diff
@@ -127,7 +127,16 @@ define([
                             14400000,
                             28800000,
                             43200000,
-                            86400000
+                            86400000,
+                            86400000 * 2,
+                            86400000 * 5,
+                            86400000 * 10,
+                            86400000 * 20,
+                            86400000 * 30,
+                            86400000 * 60,
+                            86400000 * 120,
+                            86400000 * 240,
+                            86400000 * 365
```

Comment:
Enumerating values allows us to pick increments that are human-meaningful (1 day, 2 days, 5 days, ten days) but don't follow a specific mathematical progression.